### PR TITLE
Update some SDK versions, add Java.

### PR DESCRIPTION
**Pull request by bretthoerner**
Could use a review whenever, will deploy this once I release `raven-java` `7.8.3` to Maven Central.

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -1133,13 +1133,23 @@ def get_raven_config():
 
 # TODO(dcramer): move this to sentry.io so it can be automated
 SDK_VERSIONS = {
-    'raven-js': '3.9.1',
-    'raven-python': '5.32.0',
-    'sentry-laravel': '0.5.0',
-    'sentry-php': '1.6.0',
+    'raven-java': '7.8.3',
+    'raven-java:android': '7.8.3',
```
Comment (mattrobenolt) [inline src/sentry/conf/server.py]: These are all from the same package, right? If so, I wonder if we can support this pattern more explicitly, and assume `raven-java:<thing>` is all the same?

Comment (bretthoerner) [inline src/sentry/conf/server.py]: I thought about that, but it's also nice that I can then link to the docs for each explicitly. If that's not worth it then I can look into regexing it. I sort of figured this is still a temporary solution anyway.

**Review by mattrobenolt — APPROVED** [review]
I didn't check if any of these versions were right.

**Comment by bretthoerner** [comment]
Oh dang, you approved and I did this, any thoughts on whether to keep or no? https://github.com/getsentry/sentry/pull/5029/commits/51ab09bc7aada58ef2d664b686582861c8e0961b

File: src/sentry/interfaces/sdk.py [hunk 2]
```diff
@@ -61,6 +61,23 @@ def get_api_context(self):
                 # when this is correct we can make it available
                 # 'version': newest_version,
                 'isNewer': is_newer,
-                'url': settings.SDK_URLS.get(newest_name),
+                'url': self.get_with_prefix(settings.SDK_URLS, newest_name),
             },
         }
+
+    @staticmethod
```
Comment (mattrobenolt) [inline src/sentry/interfaces/sdk.py]: I'm ok with this, but we shouldn't make this a staticmethod. There's no value in tying it to the class here. Just write a function at the module level.

**Pull request merged**